### PR TITLE
added css file

### PR DESCRIPTION
**Pull request by oldosamaibraheem11**
Change list:

1. Added css file in a folder name static 
2. Added javascript code to turn results red if error occurs 

This was done by Osama and Raju

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -7,58 +7,55 @@
...
-                                                <sup> English | Spanish | French | German | Japanese </sup></pre>
-		</h1>
-		<div style="text-align:center">
-			<form method="POST" action="">
+		<div id = "header">
+			<h1 id = "appname">App name</h1>
+			Translate Page:
+			<select id="page_language">
+					<option>English</option>
+					<option>France</option>
```
Comment (pnklshh) [inline templates/index.html]: This should French instead of France

File: templates/index.html [hunk 2]
```diff
@@ -7,58 +7,55 @@
...
+					{% for item in sentence_List %}
+					<li>{{item}}</li>
+					{% endfor %}
+				  </ol>
 		</div>
 	</body>
 	<script>$(function() {$("#language").on("change", function(event){$.ajax({data: {language : $("#language").val()},type:"POST",url : "{{url_for('Query')}}", success: function(response) {$("#partOfSpeech").html(response);;;
-      }})});});</script>
+	  }})});});</script>
+	  <script>if (document.getElementById("error").innerHTML.includes("c")) {
```
Comment (pnklshh) [inline templates/index.html]: What is 'c' ? Can we add a comment here ?

**Review by pnklshh — CHANGES_REQUESTED** [review]
I am not seeing main.css file here. Please add that to the commit and a couple of minor changes.

File: templates/index.html (hunk 2 above)
Comment (myadlapalli) [reply templates/index.html]: Does it print the alphabet c?

File: logic.py [hunk 3]
```diff
@@ -45,4 +45,7 @@ def wordcreator(word,partofspeech): # this function takes word selected and part
...
.../be no/at doubt/nn ,/, the/at Boston/np of/in that/dt era/nn could/md be/be exquisitely/ql cruel/jj in/in enforcing/vbg its/pp$ canons/nns of/in behavior/nn ./.The/at gentle/jj Channing/np ,/, revered/vbn by/in all/abn Bostonians/nps ,/, orthodox/jj or/cc Unitarian/jj ,/, wrote/vbd to/in a/at friend/nn in/in Louisville/np that/cs among/in its/pp$ many/ap virtues/nns Boston/np did/dod not/* abound/vb in/in a/at tolerant/jj spirit/nn ,/, that/cs the/at yoke/nn of/in opinion/nn crushed/vbd individuality/nn of/in judgment/nn and/cc action/nn :/: ``/`` No/at city/nn in/in the/at world/nn is/bez governed/vbn so/ql little/rb by/in a/at police/nn ,/, and/cc so/ql much/rb by/in mutual/jj inspections/nns and/cc what/wdt is/bez called/vbn public/jj sentiment/nn ./.We/ppss stand/vb more/rbr in/in awe/nn of/in one/cd another/dt than/cs most/ap people/nns ./.Opinion/nn is/bez less/ql individual/jj or/cc runs/vbz more/rbr into/in masses/nns ,/, and/cc often/rb rules/vbz with/in a/at rod/nn of/in iron/nn ''/'' ./.Even/ql more/ql poignantly/rb ,/, and/cc with/in the/at insight/nn of/in a/at genius/nn ,/, Channing/np added/vbd --/-- remember/vb ,/, this/dt is/bez Channing/np ,/, not/* Parker/np !/."
```
Comment (pnklshh) [inline logic.py]: Can we remove these unnecessary lines. I suppose this line was added for testing

File: static/main.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+#header{
+    border-style: solid ;
+    border-color: black;
+    background-color:lightblue;
+    width: auto;
+    margin-bottom: 3%;
+    height: 50px;
```
Comment (pnklshh) [inline static/main.css]: We should change height to some % instead of px since we have used % throughout the file

**Review by pnklshh — CHANGES_REQUESTED** [review]
A couple of minor changes - Added comments to both the commits

File: logic.py (hunk 3 above)
Comment (oldosamaibraheem11) [reply logic.py]: Yes, I will take that out

File: static/main.css (hunk 4 above)
Comment (oldosamaibraheem11) [reply static/main.css]: Got it

File: templates/index.html (hunk 1 above)
Comment (oldosamaibraheem11) [reply templates/index.html]: Correct

File: templates/index.html (hunk 2 above)
Comment (oldosamaibraheem11) [reply templates/index.html]: This is error code has c in it then raise error. I am going change it to corpus

**Review by pnklshh — COMMENTED** [review]
Looks good !

**Pull request merged**